samples/client/petstore/java/webclient/README.md

```diff
@@ -250,12 +249,6 @@ Authentication schemes defined for the API:
 ### bearer_test
 
 
-- **Type**: HTTP basic authentication
-
-<a id="http_signature_test"></a>
-### http_signature_test
-
-
 - **Type**: HTTP basic authentication
```

Comment:
note for discussion: when http_signature_test is kept, the samples contain this incorrect line of documentation